index.html

```diff
@@ -0,0 +1,94 @@
+            <span class="material-symbols-outlined">
+                pets
+            </span>
+            <a href="contact.html">Contact</a>
+            <span class="material-symbols-outlined">
+                pets
+            </span>
+            <a href="adopt.html">Adopt</a>
+            <span class="material-symbols-outlined">
+                pets
+            </span>
+            <a href="#">Help us</a>
+        </nav>
+    </div>
+    <div class="intro">
+        <h2>4 Paws Animal Shelter</h2>
+        <p>Where Love Finds a Home. We're here to rescue, care for, and connect pets with their forever families. Join
+            us in giving every paw a chance at a brighter future.
+            <a class="donate" href="#">Donate</a>
+
+        </p>
+    </div>
+    <div class="description">
+        <div class="first-box">
+            <p>In the heart of our community lies a beacon of hope and compassion for animals in need—4Paws Animal
+                Shelter.
+                Our shelter's history is a testament to the unwavering dedication of individuals who shared a common
+                dream of creating a safe haven for animals. Here is the heartwarming story of our shelter's journey.
+            </p>
+            <br><img src="pictures/catei.jpg" alt="dogs" class="doggies">
+        </div>
+        <div class="second-box">
+            <p> Our story begins in 2017, when a group of passionate animal lovers came together with a shared vision:
+                to provide refuge and care for homeless and mistreated animals. These pioneers recognized the pressing
+                need
+                for
+                a shelter that would offer compassion, support, and hope to those animals that had nowhere else to turn.
+            </p>
+            <br>
+            <img src="pictures/Volunteer1.jpg" alt="volunteer" class="doggies">
+        </div>
+        <div class="third-box">
+
+            <p>In the early days, 4 Paws operated from a small, rented space with limited resources.
+                Volunteers poured their time and energy into transforming this modest facility into a haven for animals.
+                They
+                worked tirelessly to rescue, care for, and find loving homes for the animals in their care. </p> <br>
+
+            <img src="pictures/dog.jpg" alt="dog" class="doggies">
+        </div>
+        <div class="fourth-box">
+            <p> Over the years, 4 Paws continued to grow and evolve. The shelter acquired a larger,
+                purpose-built facility, complete with veterinary services, dedicated kennels, and spacious play areas
+                for
+                the
+                animals. This expansion allowed the shelter to rescue and rehabilitate even more animals, transforming
+                it
+                into a
+                vital resource for the community.</p>
+            <img src="pictures/shelter.jpg" alt="shelter dogs"class="doggies">
```

Comment:
Trebuie spații între atribute.

```suggestion
            <img src="pictures/shelter.jpg" alt="shelter dogs" class="doggies">
```